quinn-proto/src/connection.rs

```diff
@@ -2598,7 +2598,13 @@ where
     ///
     /// This does not ensure delivery of outstanding data. It is the application's responsibility to
     /// call this only when all important communications have been completed, e.g. by calling
-    /// `finish` on outstanding streams and waiting for the corresponding `StreamFinished` event.
+    /// [`Connection::finish`] on outstanding streams and waiting for the corresponding
+    /// [`StreamFinished`] event.
+    ///
+    /// If [`Connection::send_streams`] returns 0, all outstanding data has been delivered. There
```

Comment:
```suggestion
    /// If [`Connection::send_streams`] returns 0, all outstanding stream data has been delivered. There
```

e.g. datagrams may not have been delivered.